fixtures/pytest_store.py

```diff
@@ -207,3 +207,8 @@ def write_line(line, **kwargs):
         # resume capturing
         with diaper:
             store.capturemanager.resumecapture()
+
+
+def factory(cls, appliance_context=None, *args, **kwargs):
```

Comment:
all objects that use "current_appliance" instead of the underlying object since that will switch the appliance under them